core/server/data/schema/fixtures/fixtures.json

```diff
@@ -44,24 +44,28 @@
             "name": "Role",
             "entries": [
                 {
-                    "name":             "Administrator",
-                    "description":      "Administrators"
+                    "name": "Administrator",
+                    "description": "Administrators"
                 },
                 {
-                    "name":             "Editor",
-                    "description":      "Editors"
+                    "name": "Editor",
+                    "description": "Editors"
                 },
                 {
-                    "name":             "Author",
-                    "description":      "Authors"
+                    "name": "Author",
+                    "description": "Authors"
                 },
                 {
-                    "name":             "Contributor",
-                    "description":      "Contributors"
+                    "name": "Contributor",
+                    "description": "Contributors"
                 },
                 {
-                    "name":             "Owner",
-                    "description":      "Blog Owner"
+                    "name": "Owner",
+                    "description": "Blog Owner"
+                },
+                {
+                    "name": "Admin API Client",
```

Comment:
For my understanding:

I generate a new api key for an integration as a logged in user.
The api key get's this role attached and not my role. If i use the api key in the integration, the permissions are limited to this role.

Isn't it weird calling a role "Admin API Client"? 🤔 
Shouldn't it be called "Integration" or similar? 🤔 

Maybe i am wrong :D

Reply:
This role is only applicable to an external client using the Admin API, an Integration can have both an Admin API key and a Content API key and this role should definitely never be applied to a Content API key.

Reply:
"Admin Integration" ;)

The perspective is from the integration and not who created the api key. 
The integration requests Ghost with an api key and it's role has certain permissions for this integration. So why calling it API Client? 🤔